lit-cli/client.py

```diff
@@ -0,0 +1,149 @@
+import click
```

Comment:
why not use argparse? How is this better?

you can use `mutually_exclusive_group` in argparse and remove all the checks you have below

Reply:
There are a few advantages: http://click.pocoo.org/5/why/#why-not-argparse